src/ir/effects.h

```diff
@@ -715,6 +720,11 @@ class EffectAnalyzer {
       if (parent.tryDepth == 0) {
         parent.throws_ = true;
       }
+    }
```

Comment:
This seems to have truncated Rethrow's handling?

Reply:
No that was intentional. So current `rethrow` doesn't have an argument that can be null, so we shouldn't have set `implicitTrap`.  I think this code was added when we had the `exnref` `rethrow` had an `exnref` as an argument (pre-Sep 2020), and then not deleted after we switched to the version of `rethrow` that takes an immediate.

Reply:
I see, thanks! Makes sense.